package-kubeapps-version.sh

```diff
@@ -132,7 +135,7 @@ generate_package_yaml() {
   # imgpkgBundle, we may need to update to a sha at some point.
   ytt -f "$template_dir/package.yaml" \
     --data-value-yaml version="$version$packaging_version_suffix" \
-    --data-value-yaml releasedAt="$(date --utc +'%Y-%m-%dT%H:%M:%SZ')" \
+    --data-value-yaml releasedAt="$(date -u +'%Y-%m-%dT%H:%M:%SZ')" \
```

Comment:
`--utc` does not work for MacOS's `date`, but `-u` works for both Linux and MacOS AFAIK.